authentik/root/settings.py

```diff
@@ -188,11 +188,16 @@ def j_print(event: str, log_level: str = "info", **kwargs):
     "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
 }
 
+REDIS_PROTOCOL_PREFIX = "redis://:"
+if bool(CONFIG.y("redis.tls")):
+    REDIS_PROTOCOL_PREFIX = "rediss://:"
```

Comment:
could be reformatted as `REDIS_PROTOCOL_PREFIX = "rediss://:" if CONFIG.y_bool("redis.tls", False) else "redis://:"` to prevent the coverage warnings

Reply:
I assume you mean `CONFIG.y_bool` and not merging those two lines into a single one?